README.md

```diff
@@ -1,71 +1,86 @@
+<img align="right" src="images/logo.svg" alt="SimpleX logo" height="90">
+
 # SimpleX chat
 
+## Federated, private, secure, decentralized
+
 [![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
 [![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-chat)](https://github.com/simplex-chat/simplex-chat/releases)
 
-## Federated chat - private, secure, decentralized
+SimpleX chat objective: to give you the full control of your chat with free open-source protocol and software of simple to deploy servers and simple to use clients providing complete privacy, security and ownership of your contacts and chat.
+
+SimpleX chat prototype is a _thin_ terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker that uses [SMP protocol](https://github.com/simplex-chat/simplexmq/blob/master/protocol/simplex-messaging.md) and [SMP agent protocol](https://github.com/simplex-chat/simplexmq/blob/master/protocol/agent-protocol.md).
```

Comment:
I liked `thin` in quotes more, but now I think it'd be best to remove any style altogether, "thin client" is common enough not to stress this word, so I suggest just
```SimpleX chat prototype is a thin terminal UI```